### PR TITLE
Edit models to be auto-compatible with 128 x 128 images

### PR DESCRIPTION
**Pull request by janghyuk-choi**
Attention() and ComponentVAE() in MONetModel behave by referring to the argument `load_size` which stands for the input image size.

### CODE REVIEW

**Comment by baudm** [comment]
I see that you've added a switch to determine the blocks used at runtime. For both image sizes, the architectures will be the same but for 64x64, `downblock5` and `upblock2` will be unused. Doesn't PyTorch complain about some weights not having gradients during training? I don't have an environment to test this right now.

**Comment by janghyuk-choi** [comment]
@baudm 
Thank you for pointing it out!
Though the problem did not occur in my case, I totally agree that dangling tensors must not exist and may result in problems.

I fixed it by adding a switch for the initialization of `downblock5` and `upblock2`.
4f341adc8218e4340b2bbdae234e7d7f275f011c
